FEATURES.md

```diff
@@ -0,0 +1,175 @@
+# Features
+
+This document describes a list of use-cases and features that this library provides.
+
+## Use Cases
+
+The following are the core use cases that this integration supports. If you have another use case, please let us know by creating an Issue.
+
+* UC 1: As a user, I want to trigger a Keptn quality gate evaluation from within Jenkins
+* UC 2: As a user, I want to trigger a delivery with Keptn from within Jenkins
```

Comment:
I would not be too limited. Please broaden the scope of the use-case to: 

* UC 2: As a user, I want to execute (trigger) an automation sequence with Keptn from within Jenkins. 

--- 

What we see is that many automation sequences are built for other purposes, and not only `delivery`.

Reply:
I agree, but that's the Use Case that's currently supported. We cannot trigger an arbitrary sequence right now. That would be a separate use-case that we should define and implement.

Reply:
Just asking, is it not possible from an implementation point of view to define the sequence we would like to trigger?

Reply:
Currently the implementation contains functions to trigger respective sequences with a payload, but it's not possible to change the name of the sequence.

* `sendDeliveryTriggeredEvent`-> `"sh.keptn.event.${stage}.delivery.triggered"`
* `sendStartEvaluationEvent` -> `"sh.keptn.event.${stage}.evaluation.triggered"`

Reply:
I will dismiss this comment, as, again, we are showing the current implementation, and not the desired state.